include/glow/Base/Tensor.h

```diff
@@ -347,6 +348,26 @@ class Tensor final {
     return *this;
   }
 
+  /// Dump a textual representation of the Tensor into provided output stream.
+  void dump(llvm::raw_ostream &os) const;
```

Comment:
For Tensors, we may want to additionally provide a couple more dump functions which are similar to `Handle::dump` versions that take an `unsigned maxNumElem` parameter. This makes it easier to dump as many elements of the tensor as you need.

Reply:
> For Tensors, we may want to additionally provide a couple more dump functions which are similar to `Handle::dump` versions that take an `unsigned maxNumElem` parameter. This makes it easier to dump as many elements of the tensor as you need.

OK.